### PR TITLE
Feat: Create README.md for notification service

### PR DESCRIPTION
**Pull request by ZixMai**
1.  Created README.md for notification service

### CODE REVIEW

**Review by Nhsdkk — APPROVED** [review]

File: notification_service/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+       - Установка осуществляется через **docker**, для установки пакета микросервиса введите и выполните команду:
+		```
+    	docker pull ghcr.io/unspokenteam/notification_service:latest
+ 		```
+       - Создайте .env файл, укажите его путь в команде и выполните её:
+		```
+		docker run -d -p 8083=8083 —env-file $PATH_TO_ENV ghcr.io/unspokenteam/notification_service:latest
+ 		```
+
+	2. **Через poetry**
```
Comment (Nhsdkk) [inline notification_service/README.md:44]: Add proto and prisma client generate steps

File: notification_service/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+___
+# Notification service
+[![Build notification service](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_notification_service.yaml/badge.svg)](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_notification_service.yaml)
+___
+**Установка и описание**
+___
+1. **Образец .env файла**
+	```
+	DATABASE_URL="DATABASE_URL"
```
Comment (Nhsdkk) [inline notification_service/README.md:10]: Add comments for variables

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

File: notification_service/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+	ENVIRONMENT="ENVIRONMENT"
+	```
+ 	- где:
```
Comment (Nhsdkk) [inline notification_service/README.md]: Move this to comments inside dotenv example

File: notification_service/README.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      - DATABASE_URL - адрес базы данных
+      - ENVIRONMENT - вид окружения
+___
+2. **Установка**:
+	- Доступно несколько способов установки.
```
Comment (Nhsdkk) [inline notification_service/README.md]: Add pre-installation step to create db

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

**Review by MaxPlays35 — APPROVED** [review]

**Review by Nhsdkk — APPROVED** [review]
LGTM

**Pull request merged**